### PR TITLE
[MRG + 1] count shapes utility

### PR DESCRIPTION
**Pull request by kowey**
Will leave this at your discretion to figure out if this is useful for something like stac-util or should be more properly left in some iPython notebook somewhere and forgotten :-)

### CODE REVIEW

File: educe/stac/util/cmd/count_shapes.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    """Return the outgoing edges for a given node"""
+    return [e for e in gra.links(node)
+            if gra.is_relation(e) and gra.rel_links(e)[0] == node]
+
+def _maybe_lozenge(gra, node):
+    """Return (if applicable) lozenge nodes/edges starting from the given node
+
+    If the given node looks like the start of a lozenge, return all of
+    the nodes participating in the lozenge. If not, return None
+
```
Comment (moreymat) [inline educe/stac/util/cmd/count_shapes.py]: Could I bother you into explicitly documenting parameters? I think I get the type of gra right now but fear future me might not like it so much.

**Comment by moreymat** [comment]
Otherwise, LGTM.

File: educe/stac/util/cmd/count_shapes.py (hunk 1 above)
Comment (kowey) [inline educe/stac/util/cmd/count_shapes.py]: Well OK for this function, but I don't think I'm going to want to track down all my uses of the `gra` convention and Parameter those

Comment (moreymat) [inline educe/stac/util/cmd/count_shapes.py]: One mention here should be enough for future reference :-)

**Pull request merged**